android/src/com/mapswithme/maps/MwmActivity.java

```diff
@@ -782,6 +788,11 @@ protected void onRestoreInstanceState(@NonNull Bundle savedInstanceState)
 
     if (!mIsFragmentContainer && RoutingController.get().isPlanning())
       mRoutingPlanInplaceController.restoreState(savedInstanceState);
+
+    if(mNavigationController != null)
+    {
+      mNavigationController.restoreState(savedInstanceState.getBoolean(STATE_SHOW_TIME_LEFT));
```

Comment:
The same here. Please pass the whole state bundle to the `mRoutingPlanInplaceController.restoreState(@NonNull Bundle savedState)`